build/docker/centos/Dockerfile

```diff
@@ -74,10 +74,8 @@ RUN yum install -y \
       perl-Bit-Vector \
       perl-Class-Accessor \
       perl-ExtUtils-MakeMaker \
-      perl-Test-Simple \
       perl-IO-Socket-SSL \
-      perl-Net-SSLeay \
-      perl-Crypt-SSLeay
```

Comment:
I have reverted this change locally, since both SSLeay modules are needed for SSL/TLS sockets and for Http(s)Client transport.  When I merge upstream I will drop the changes to this file.